portfolio/src/main/webapp/style.css

```diff
@@ -81,3 +81,22 @@ button{
 button:hover{
     cursor: pointer;
 }
+
+form{
```

Comment:
"Use a space between the last selector and the declaration block."

https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation